brmp/pyro_backend.py

```diff
@@ -3,20 +3,20 @@
 from sys import stderr
 
 import numpy as np
+import torch
+
 import pyro
 import pyro.poutine as poutine
-import torch
+from brmp.backend import Backend, Model
+from brmp.fit import Samples
+from brmp.pyro_codegen import gen
+from brmp.utils import flatten, unflatten
 from pyro.infer import SVI, Trace_ELBO
 from pyro.infer.autoguide import AutoMultivariateNormal
```

Comment:
I used `make format` which gave me this, but we can have a separate treatment for brmp related imports in `setup.cfg` (i.e. previous version), if that looks better in a separate PR.